packages/nimble-components/src/rich-text/editor/tests/rich-text-editor-mention.spec.ts

```diff
@@ -849,4 +831,550 @@ describe('RichTextEditor user mention via template', () => {
             pageObject.getEditorMentionViewAttributeValues('mention-label')
         ).toEqual(['John Doe', 'Mary Wilson']);
     });
+
+    describe('user mention button', () => {
+        beforeEach(async () => {
+            await waitForUpdatesAsync();
+        });
+
+        it('should have `at icon` button', () => {
```

Comment:
This test is essentially duplicated above in the 'user mention button' describe block. I think you can get rid of one of the copies.

Reply:
Removed.